lib/Player.js

```diff
@@ -0,0 +1,16 @@
+class Player {
+  constructor(name, score) {
+    this.name = name;
+    this.score = 0;
+  }
+  
+  increaseScore(){} 
+ 
+  
+  decreaseScore() {}
```

Comment:
Can you combine the increase and decrease score methods? If we pass another argument, a boolean if the player was right or wrong, then we can use that to determine if score should be increased or decreased.